libqtile/layout/verticaltile.py

```diff
@@ -118,21 +124,21 @@ def configure(self, window, screen_rect):
             index = self.clients.index(window)
 
             # border
-            if n > 1:
-                border_width = self.border_width
-            else:
-                border_width = 0
+            border_width = self.border_width if n > 1 else self.single_border_width
```

Comment:
Might also be worth collapsing these if statements, border, margin, width and height

Reply:
done

Reply:
If you're talking about the ```__init__``` function, I was trying to keep it same as MonadTall

<img width="960" alt="image" src="https://user-images.githubusercontent.com/71731492/201539817-5c940003-4bc5-4ec4-b88f-fb351f3065a4.png">